lapis/drone.py

```diff
@@ -104,23 +113,27 @@ def _add_resources(self, keys: list, target: dict, source: dict, alternative_sou
             raise ResourcesExceeded()
 
     @staticmethod
-    def _remove_resources(keys: list, target: dict, source: dict, alternative_source: dict):
+    def _remove_resources(keys: list, target: dict, source: dict,
+                          alternative_source: dict):
         for resource_key in keys:
             try:
                 target[resource_key] -= source[resource_key]
             except KeyError:
                 target[resource_key] -= alternative_source[resource_key]
 
-    async def start_job(self, job: Job, kill: bool=False):
+    async def start_job(self, job: Job, kill: bool = False):
         """
         Method manages to start a job in the context of the given drone.
-        The job is started independent of available resources. If resources of drone are exceeded, the job is killed.
+        The job is started independent of available resources. If resources of
+        drone are exceeded, the job is killed.
 
         :param job: the job to start
-        :param kill: if True, a job is killed when used resources exceed requested resources
+        :param kill: if True, a job is killed when used resources exceed
+        requested resources
```

Comment:
See above note on aligning in sphinx docstrings.

```suggestion
                     requested resources
```